### PR TITLE
Enable the insertion of svg diagrams into pdf files (latex & beamer), converting them into png images via inkscape

### PR DESCRIPTION
**Pull request by fermarsan**
This contribution enables the insertion of svg diagrams into pdf files (latex & beamer), converting them into png images via inkscape.

### CODE REVIEW

File: _extensions/quarto-svgbob/_extension.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 title: quarto-svgbob
 author: Dheepak Krishnamurthy
-version: 1.0.0
+  # - Fernando Martínez Santa
```
Comment (kdheepak) [inline _extensions/quarto-svgbob/_extension.yml]: Do you want to use the authors block here instead?

```
authors:
  - name: Dheepak Krishnamurthy
  - name: Fernando Martinez Santa
 ```

That way you'll get credit for it on the quarto website.

File: _extensions/quarto-svgbob/_extension.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+version: 1.0.1
```
Comment (kdheepak) [inline _extensions/quarto-svgbob/_extension.yml]: Probably worthy of v2.0.0

File: _extensions/quarto-svgbob/quarto-svgbob.lua [hunk 3]
```diff
@@ -42,15 +44,36 @@ function Render(elem)
...
 end
 
+function InsertSvgLatex(svg_data)
+	local name = 'assets/'
+	local isok, errstr, errcode = os.rename(name, name)
+	if isok == nil then
+		if errcode ~= 13 then
+			os.execute("mkdir assets")
+		end
+	end
```
Comment (kdheepak) [inline _extensions/quarto-svgbob/quarto-svgbob.lua]: Is it worth copying over the `os.exists` and `os.mkdir` functions (with a credit to the source) from here?

https://github.com/mokeyish/obsidian-enhancing-export/blob/16cdb17ef673e822e03e6d270aa33b28079774cc/lua/polyfill.lua#L25-L47

**Comment by kdheepak** [comment]
Thanks for making a PR! This looks good to me! I made minor comments.

File: _extensions/quarto-svgbob/_extension.yml (hunk 1 above)
Comment (fermarsan) [reply _extensions/quarto-svgbob/_extension.yml]: sounds good..

File: _extensions/quarto-svgbob/_extension.yml (hunk 2 above)
Comment (fermarsan) [reply _extensions/quarto-svgbob/_extension.yml]: ok

File: _extensions/quarto-svgbob/quarto-svgbob.lua (hunk 3 above)
Comment (fermarsan) [reply _extensions/quarto-svgbob/quarto-svgbob.lua]: I will check it....

**Comment by fermarsan** [comment]
Thanks, I will make changes and do a new PR.

**Comment by fermarsan** [comment]
Hello, I already did the suggested changes, but for a weird reason the file `_extension.yml` doesn't allow more than one author. 

Is it necessary to do a new PR or you can check directly the changes?

File: _extensions/quarto-svgbob/_extension.yml [hunk 4]
```diff
@@ -1,7 +1,8 @@
 title: quarto-svgbob
-author: Dheepak Krishnamurthy
-  # - Fernando Martínez Santa
-version: 1.0.1
+author:
```
Comment (kdheepak) [inline _extensions/quarto-svgbob/_extension.yml]:
```suggestion
authors:
```

Comment (kdheepak) [reply _extensions/quarto-svgbob/_extension.yml]: If you make it `authors` it should work fine

Comment (fermarsan) [reply _extensions/quarto-svgbob/_extension.yml]: Thanks a lot. I already did this change.

**Comment by fermarsan** [comment]
I already did all the suggested changes and include the possibility to call `svgbob` in addition to `svgbob_cli`.

**Comment by kdheepak** [comment]
Thanks for making the PR! LGTM!

**Pull request merged**